### ISSUE TITLE
Add link from publiccode.net to blog.publiccode.net

### ISSUE THREAD

**Issue by ElenaFdR** [publiccodenet/publiccode.net#26]
Only do once blog.publiccode.net is ready to share.

**Comment by ElenaFdR** [comment]
[PR #15](https://github.com/publiccodenet/jekyll-theme/pull/15) should fix this.

**Comment by ElenaFdR** [comment]
See discussion on PR for debate about the approach.

**Comment by Ainali** [comment]
Based on the discussions, should this be closed with a "Won't fix" or are we adding a link somewhere but less prominent?

**Comment by ElenaFdR** [comment]
I'm closing this because we're not planning to do this on its own. Our [blogging guidance](https://about.publiccode.net/activities/communication/blogging.html) explains why we've chosen not to link to the blog on its own, preferring instead to link to specific blogposts.
